### DISCUSSION TITLE
ping failing

### DISCUSSION THREAD

**Discussion by djhurlburt** [NightscoutFoundation/xDrip#2632] [Q&A]
I've been following https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud to the "T" and get to "Phase 2". The system seems to be trying to ping my subdomain that I created but it continuously fails. After 12 tries I see it try to confirm connection to 'freedsn.afraid.org' and that seems to work, but then it tries to resume connection to the subdomain and 100% packet loss. This keeps going 6 more times (gets to trial number 18) and then it once again "resolving freedns.afraid.org" which seems to work, then it once again tries to ping the subdomain several more time and then I get a screen that says:
"Developed by the xDrip team
Please close this window. Open a new SSH terminal. Run FreeDNS Setup again to complete FreeDNS setup"

I never get to this step:

> After successful setup, you will be asked to enter an email address for security notices.
> 
> Enter an email address.

I tried pinging the ipaddresses from my personal computer and get to both of them, but the connection from the GoogleServer to this subdomain seems to be failing.

**Comment by Navid200** [comment]
Please give me 5 minutes.

**Comment by Navid200** [comment]
Would you please confirm that you intend to use FreeDNS or not.
Thanks

**Reply by djhurlburt** [reply]
I am using FreeDNS.  I tried making various subdomains, but that doesn't seem to help since they are the same ip.

The SSH-in-browser shows:
--2023-01-16 18:10:27--  https://freedns.afraid.org/dynamic/update.php?RVdoVUhLQzhWaWdWRExoRkZiZzFoVVczOjIxMDMyMzEz
Resolving freedns.afraid.org (freedns.afraid.org)... 69.42.215.252
Connecting to freedns.afraid.org (freedns.afraid.org)|69.42.215.252|:443... connected.
HTTP request sent, awaiting response... 200 OK

**Reply by Navid200** [reply]
OK  

Would you please post a screenshot of the status page.  That in itself will not show anything private.  
Please don't post anything that shows any IDs or passwords.

**Reply by djhurlburt** [reply]
![Capture](https://user-images.githubusercontent.com/122819185/212743294-911d376a-a45c-404e-8669-093acbd0ab14.JPG)

**Reply by Navid200** [reply]
You are posting things that include your hostnames.  I wouldn't do that.  

Can we go one step at a time please?
Would you please post a screenshot of the status page?

**Reply by djhurlburt** [reply]
Well, you asked for a screen shot and I don't see username or pwd, so I thought it was safe!

Status page?  What status page?  what is your definition of 'status page'?

**Reply by djhurlburt** [reply]
are you talking about selection "1"?

**Reply by Navid200** [reply]
We have just had a new release.
I will need to update the guide to correspond to it.

But, I would like to help you first.  
When you open a terminal, you should see a menu.  
Do you mind posting a screenshot of it here?

**Reply by djhurlburt** [reply]
![Capture](https://user-images.githubusercontent.com/122819185/212744290-fe529757-4e21-4e9f-be67-912afece8815.JPG)

**Reply by djhurlburt** [reply]
![Capture](https://user-images.githubusercontent.com/122819185/212744449-6e94f142-bee9-4ee5-967d-5f72fcf8e53b.JPG)

**Reply by Navid200** [reply]
How many subdomains do you have?
You can have a total of 5 for free.
Can you create a new one?

**Reply by djhurlburt** [reply]
I currently have two - but they both seems to have the same ip address.  I can create another.

**Reply by Navid200** [reply]
The ip address is defined by the machine.  The job of the subdomain is to map a name to the ip of your machine.

**Reply by djhurlburt** [reply]
okay I created another subdomain.  What would you like me to do next?

**Reply by Navid200** [reply]
Please run the FreeDNS utility with it. 
If it fails again, please let me know.

**Reply by djhurlburt** [reply]
just to confirm, the "FreeDNS utility" is step 3 (phase 2) or something else?

**Reply by djhurlburt** [reply]
or is it step 10 "FreeDNS Setup" ?

**Reply by djhurlburt** [reply]
well, I tried step 10 "FreeDNS Setup" and I'm getting the same '100% packet loss' issue

**Reply by djhurlburt** [reply]
oh, wait, the 2nd time it was able to go through

**Reply by Navid200** [reply]
That's great.

**Reply by djhurlburt** [reply]
![Capture](https://user-images.githubusercontent.com/122819185/212748798-24a7d638-a576-4258-865e-f19e4050fdd4.JPG)

**Reply by Navid200** [reply]
That looks good to me.

Now, open a web browser and enter the hostname it it.
It should ask you to authenticate ...

**Reply by djhurlburt** [reply]
and...my hostname is the subdomain?

**Reply by Navid200** [reply]
It's more than that.
If you go to the main menu and go to the status page, at the bottom, you will see 2 options.  One of them shows you your password and hostname.
